mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -260,11 +260,49 @@ void CSaveLocSystem::LevelInitPreEntity()
     }
 }
 
+bool CSaveLocSystem::LoadStartMarks()
+{
+    if (m_pSavedLocsKV)
+    {
+        DevLog("Loaded startmarks from %s!\n", SAVELOC_FILE_NAME);
+
+        if (!m_pSavedLocsKV->IsEmpty())
+        {
+            KeyValues *kvMapSavelocs = m_pSavedLocsKV->FindKey(gpGlobals->mapname.ToCStr());
+            if (kvMapSavelocs && !kvMapSavelocs->IsEmpty())
+            {
+                KeyValues *kvStartMarks = kvMapSavelocs->FindKey("startmarks");
+                if (!kvStartMarks)
+                    return false;
+
+                CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
+
+                if (!pPlayer)
+                    return false;
+
+                FOR_EACH_SUBKEY(kvStartMarks, kvStartMark)
+                {
+                    int track = atoi(kvStartMark->GetName());
```

Comment:
Use `Q_atoi` here, it's a string-specific thing that doesn't rely on the STL.

Reply:
I wasn't aware of this, thanks. atoi has now been replaced with Q_atoi.